nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/FormModel.java

```diff
@@ -2,12 +2,13 @@
 
 import java.beans.PropertyChangeListener;
 import java.beans.PropertyChangeSupport;
+import java.io.Serializable;
 
 /**
  * <code>FormModel</code> is an abstract class that represents the backing data for a form. It provides a mechanism
  * for form elements to retrieve their values to display to the user and persist changes to the model upon changes.
  */
-public abstract class FormModel {
+public abstract class FormModel implements Serializable {
```

Comment:
I wonder if there's another way we can maintain the model instance on rotations besides implementing the `Serializable` interface. It's reasonable, but my main concern is that it might be a bit too restricting for users if they need to implement their own model. This essentially means their model and its class members must be `Serializable`.

It might be a reasonable restriction of there's no other way, but I'd like to make sure we explore other options too. Were there other ideas you considered?